src/core/components/RichTextEditor/AmityAdapterEditor.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactNode, useLayoutEffect } from 'react';
+
+import RichTextEditor, { useEditor } from './components/Editor';
+import { markdownToSlate, slateToMarkdown } from './markdownParser';
+import { EditorValue } from './models';
+import { MentionOutput } from './plugins/mentionPlugin/models';
+import { stripMentionTags } from './plugins/mentionPlugin/utils';
+
+export type AmityAdapterProps = {
```

Comment:
Amity is the Social network provider we are using to build our Community. And this repo is the for of their web UI kit.
So this adapter is needed to match the API they are using for their textures. So it can easily replace them.